src/components/grid-aware/Footer/Footer.module.css

```diff
@@ -0,0 +1,257 @@
+/* Background bleed */
+
+.bleedWrapper {
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-template-rows: auto 0;
+  margin: auto;
+  max-width: var(--grid-bleed-max-width);
+}
+
+/* Main grid */
+
+.gridParent {
+  column-gap: var(--grid-column-gap);
+  display: grid;
+  grid-template-rows: 120px 380px 78px;
+}
+
+/* Override grid item default of auto so that item contents cannot stretch the grid itself */
+.gridParent > * {
+  min-width: 0;
+}
+
+@media (--tablet-and-down) {
+  .bleedWrapper {
+    text-align: center;
+  }
+
+  .gridParent {
+    column-gap: var(--grid-column-gap);
+    display: grid;
+    grid-template-rows: 190px auto auto;
+  }
+}
+
+/* Top Row */
+
+.gridTopRow {
+  align-content: center;
```

Comment:
This property has no effect on a flex parent that doesn't wrap.

Reply:
ok